GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
+Firstly, start the application by running `python3 main.py`.
+
+You will be shown the main window of the application (pictured below) which contains both a list of components and a 3D view visualising the components listed with their respective shape and position. 
+The sample component is added by default, and is indicated by the red cube. You can move around the 3D view by using the arrow keys to move, click+drag to pan, and pgup/down to zoom.
+
+![](resources/images/NeXusConstructor_001.png)
+
+By clicking the "Nexus File Layout" tab in the left pane of the window, the NeXus file layout can be shown in a tree structure.
+
+![](resources/images/Selection_002.png)
+
+## Adding components
+
+The "add component" button, found in the layout above the list of components (pictured) can be clicked to open a dialog containing possible fields and options a component may include.
```

Comment:
Assume the reader of this guide doesn't know anything about NeXus. You'll need to introduce terminology like _fields_.